cfme/fixtures/service_fixtures.py

```diff
@@ -80,4 +80,4 @@ def order_catalog_item_in_ops_ui(appliance, provider, provisioning, vm_name, dia
                                                                  partial_check=True)
     provision_request.wait_for_request()
     assert provision_request.is_finished()
-    return catalog_item.name
+    return catalog_item
```

Comment:
Are you sure this fixture is not used in other tests except `test_myservice_crud` and `test_vm_console`?

Reply:
Hey @quarckster, yes I made that sure by talking to @sshveta. She mentioned that it shouldn't be a problem.